pom.xml

```diff
@@ -938,5 +936,16 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>podman-enabled</id>
+            <activation>
+                <property>
+                    <name>podman-enabled</name>
```

Comment:
My thinking was that if you have disabled Quarkus build with `<quarkus.build.skip>${podman-disabled}</quarkus.build.skip>`, you don't need any profiles, that was my thinking. But it's just proposal, I don't have time to test it, you tell me when you test it. Thank you

Reply:
btw I propose to lookup how we use `quarkus.build.skip` elsewhere, it may help you